src/ChipVisitAstrometryMapping.cc

```diff
@@ -121,5 +121,14 @@ void ChipVisitAstrometryMapping::freezeErrorTransform() {
                       " The routine ChipVisitAstrometryMapping::freezeErrorTransform() was thought to be "
                       "useless and is not implemented (yet)");
 }
+
+std::string ChipVisitAstrometryMapping::toString() const {
+    std::stringstream out;
+    out << "Composed Astrometry Mapping:" << std::endl;
+    out << "Sensor f(x,y)->(u,v): " << _m1->toString() << std::endl;
+    out << "Visit f(u,v)->alpha,delta: " << _m2->toString() << std::endl;
+    return out.str();
+}
```

Comment:
Hmmm... should I change those `print` to be `std::ostream& print(std::ostream &stream)`, so I can put them inside the `operator<<` like `out << map.print(out) << ...`?

Reply:
I don't think that would work -- it would be equivalent to `out << out << ...`, which I'm pretty sure would not compile.

Plus the desired syntax is kind of confusing because of the double `out`.

Reply:
Alternative suggestions? Changing these to `void print(stream)` would make the above code look like this, which I'm not sure is an improvement.

```
     out << "Sensor f(x,y)->(u,v): ";
    _m1->print(out);
    out << std::endl;
```

Reply:
`ChipVisitAstrometryMapping::print` becomes idiomatic again if `_m1` and `_m2` (which I assume are `Model`s) support `operator<<`. The only downside is that you have an extra function definition elsewhere in your code.